### PR TITLE
Updated macos ledger install steps.

### LINKED ISSUE #397: Set up ledger on macOS clarifications
Two suggestions for clarifications/simplifications for the ledger setup on macOS.

1. Explain how to open a terminal in a specific folder (and where the folder most likely is located)
   - Related to step 6 in installing certificate and step 1 of installing Concordium ledger app.
   - Most likely, the folder will be `~/Downloads/concordium-ledger-app-2.0.1-target-2.0.0` (or similar)
   - So you would open a terminal and `cd ~/Downloads/concordium-ledger-app-2.0.1-target-2.0.0`
   - Then you run the relevant script.
2. Combine dependency installation steps 2,3,4 into one with `brew install python@3.9 libusb libjpeg`
   - (We could even include step 5 `brew install python@3.9 libusb libjpeg && pip3 install ledgerblue`)

### PR DESCRIPTION
**Pull request by dg-concordium**
## Purpose

To simplify and clarify the ledger install instructions for macOS.

Closes #397.

## Changes

Added steps describing how to navigate to a directory in Terminal; combined dependency installation steps into one step.

## Checklist

- [X] My code follows the style of this project.
- [X] The code compiles without warnings.
- [X] I have performed a self-review of the changes.
- [ ] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [ ] (If necessary) I have updated the CHANGELOG.

## CLA acceptance

_Remove if not applicable.

By submitting the contribution I accept the terms and conditions of the
Contributor License Agreement v1.0
- link: https://developers.concordium.com/CLAs/Contributor-License-Agreement-v1.0.pdf

- [X] I accept the above linked CLA.

### CODE REVIEW

File: source/shared/net/desktop-wallet/install-ledger-app.rst [hunk 1]
```diff
@@ -162,37 +162,17 @@ Install Concordium Ledger app on macOS
...
 
-#. Open the Terminal application. Click the **Launchpad** icon in the **Dock**, type *Terminal* in the search field, then select **Terminal**.
+#. To open the Terminal application, click the **Launchpad** icon in the **Dock**, type *Terminal* in the search field, then select **Terminal**.
 
-#. Install the package manager Homebrew. Copy the following line into the Terminal and press enter.
+#. Navigate to where you have downloaded the Ledger install package. For example, this might be ``~/Downloads/concordium-ledger-app-2.0.1-target-2.0.0``. To navigate to this directory in a Terminal, enter ``cd ~/Downloads/concordium-ledger-app-2.0.1-target-2.0.0``.
 
-   .. code-block:: console
-
-      /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```
Comment (Bargsteen) [inline source/shared/net/desktop-wallet/install-ledger-app.rst:171]: They still need to this script. That is how you download and install Homebrew :)

File: source/shared/net/desktop-wallet/install-ledger-app.rst [hunk 2]
```diff
@@ -162,37 +162,17 @@ Install Concordium Ledger app on macOS
 Install Homebrew, Python3, and pip
 ----------------------------------
 
-#. Open the Terminal application. Click the **Launchpad** icon in the **Dock**, type *Terminal* in the search field, then select **Terminal**.
+#. To open the Terminal application, click the **Launchpad** icon in the **Dock**, type *Terminal* in the search field, then select **Terminal**.
```
Comment (Bargsteen) [inline source/shared/net/desktop-wallet/install-ledger-app.rst:165]: The user might not have the Launchpad in their dock. It's just a shortcut, and they could have removed it.
And users might not know that the grid of apps is called the launchpad. (I know that I wrote this, but we might as well update it if we can come up with a better phrase 😊  )

File: source/shared/net/desktop-wallet/install-ledger-app.rst (hunk 1 above)
Comment (Bargsteen) [reply source/shared/net/desktop-wallet/install-ledger-app.rst:171]: So two steps seem reasonable.
1. Install homebrew
2. Install all the rest

**Review by Bargsteen — CHANGES_REQUESTED** [review]
A few comments, but I think the simplification is a good idea 😊

File: source/shared/net/desktop-wallet/install-ledger-app.rst (hunk 2 above)
Comment (dg-concordium) [reply source/shared/net/desktop-wallet/install-ledger-app.rst:165]: I'm not a Mac user. So what would a user need to do if Launchpad isn't in their dock? Or does Apple have a support article we could link to? That would be the best.

Comment (Bargsteen) [reply source/shared/net/desktop-wallet/install-ledger-app.rst]: Great idea with the link to Apple's own terminal guide ;)

**Review by Bargsteen — APPROVED** [review]
Great! 👍

**Pull request merged**